renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -2337,7 +2446,7 @@ rdcarray<PixelModification> VulkanReplay::PixelHistory(rdcarray<EventUsage> even
                                                        ResourceId target, uint32_t x, uint32_t y,
                                                        const Subresource &sub, CompType typeCast)
 {
-#if 1
+#if 0
```

Comment:
I think you should delete this code entirely, we'll never want to enable it again.

However you should also add a check for the API properties to see if pixel history is supported - this prevents scripts which don't check the API properties from calling into the pixel history path. The API properties allows for more graceful UI fallback/enabling but we should also code defensively here.

Reply:
Removed. I added a helper function to access the config, since RDOC_CONFIG currently can only be used in one place. Not sure if there is a better way to do this.

Reply:
If you look at how it's implemented in `vk_shaderdebug.cpp` you can use the same method to access `GetAPIProperties()` and check the bool there.

Reply:
Ahh makes sense, thanks!